### ISSUE TITLE
Outline instructions in screens of development version

### ISSUE THREAD

**Issue by JosedeKruif** [UUDigitalHumanitieslab/texcavator#140]
Instruction texts have a second line with large indention when shown on a narrow screen. Change central alignment to left alignment?

**Comment by jgonggrijp** [comment]
Yes, I can imagine that centered alignment gets hard to read at greater screen widths. Changing to left alignment is a slight improvement, but it still forces you to move your head left to right in order to read long lines. Perhaps a more significant improvement would be to restrict the width of the instruction text blocks?
